frontend/simple/assets/sass/theme/kindergarten-professional.scss

```diff
@@ -37,6 +38,7 @@ $colors: map-merge($colors, $addColors);
 
 $link: $primary;
 $text: $grey-darker;
+// $text-light: $grey; // REVIEW - create a variable for lighter texts... ?
```

Comment:
How do you currently handle light colored / lightgrey text?

Reply:
when in the DOM add `.has-text-grey`, on CSS use variable `$grey`. Once we have themes, the "lighter text might not be grey... that's why my concern about making it consistent and use `.has-text-light` and `$text-light` instead of `grey`

Reply:
All the other colors text I use primary/secondary/danger/... variables :)

Reply:
So, finally figured it out the correct way of doing this with Bulma variables only:

`$text-light` - used on text with grey color. It can also be used with class `.has-text-text-light` or `.has-background-text-light`
`$light` - the grey light color, used as background on light buttons for example. It can also be used with class `.has-text-light` or `.has-background-light`.

I've replaced all places with `.has-text-grey` for `.has-text-text-light` and texts with `$grey` for `$text-light`.